package-lock.json

```diff
@@ -1,11600 +1,8 @@
 {
   "name": "webrecorder",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Again, this should not change.

Reply:
I have deleted this file and copied the package-lock.json from main branch to this branch. Now there is no changes from this file.